.travis.yml

```diff
@@ -1,5 +1,8 @@
 sudo: false
 language: cpp
+branches:
+    only:
+        - master
```

Comment:
It doesn't. PRs are handled additionally. Have a look at the first commit in this PR, that did not have this. There are 2 builds from travis, which should not be.